src/blog/2024-11-02/index.md

```diff
@@ -0,0 +1,107 @@
+title: Pre-Alpha v10.0 Release Note
+summary: Release note for Neotokyo; Rebuild Pre-Alpha v9.0
+author: brysondev
+
+# Pre-Alpha v10.0 Release Note
+2024-11-02 brysondev
+
+* [Download v10.0-prealpha build](https://github.com/NeotokyoRebuild/neo/releases/tag/v10.0-prealpha)
+* [Install NT;RE (client)](/guide/install/)
+* [GitHub Issues (Bug reports and feature requests)](https://github.com/NeotokyoRebuild/neo/issues)
+
+## New features
+
+### Deathmatch
+
+Deathmatch is the same as TDM with both CTG capture points & ghosts disabled, and enables respawning. 
```

Comment:
Make it more clear it's free for all, win by either XP limit (default 50) or time out (default 10 mins)